brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1065,18 +1069,19 @@ void shouldSaveSuccessReportContainingIncomingHandleWhenUpdatingExistingPublicat
                    is(equalTo(nvaBrageMigrationDataGenerator.getBrageRecord().getId().toString())));
     }
 
-    //TODO: flaky test
     @Test
     void shouldPersistMergeReport()
         throws IOException, BadRequestException, nva.commons.apigateway.exceptions.NotFoundException {
         var cristinIdentifier = randomString();
         var nvaBrageMigrationDataGenerator = new NvaBrageMigrationDataGenerator.Builder().withPublishedDate(null)
-                                                 .withType(TYPE_BOOK)
+                                                 .withType(TYPE_FILM)
                                                  .withCristinIdentifier(cristinIdentifier)
```

Comment:
Fixing this flaky test. Random book with ISBN makes it difficult to create book with ISBN, as random isbn is often not valid isbn and ends up as additionalIdentifier instead of ISBN.

Reply:
Pr that fixes that:

https://github.com/BIBSYSDEV/nva-datamodel-java/pull/484